libibumad/umad.h

```diff
@@ -62,7 +62,7 @@ union umad_gid {
 	} global;
 } __attribute__((aligned(4))) __attribute__((packed));
 
-#define UMAD_MAX_DEVICES 32
+#define UMAD_MAX_DEVICES 128
```

Comment:
This is separate commit with normal explanation why it is needed and why it is safe to change.

There is no need to write "Raise UMAD max device limit to 128.". It is clear from the code.